css-view-transitions-2/Overview.bs

```diff
@@ -616,186 +563,391 @@ Prepend this to the [=Perform pending transition operations=] algorithm given a
 
-		1. [=Assert=]: |document| [=has been revealed=] is true.
+An '':active-view-transition-type()'' pseudo-class matches the [=document element=] when its [=node document=] has an non-null [=active view transition=],
+whose [=ViewTransition/active types=] [=list/contains=] at least one of the <<custom-ident>> arguments.
 
-		1. [=Update the opt-in state for outbound transitions=] for |document|.
+## Activating the transition type using JavaScript ## {#types-js}
 
-		1. Let |inboundViewTransitionParams| be |document|'s [=inbound view transition params=].
+### Extending {{Document/startViewTransition|document.startViewTransition()}} ### {#extend-document-types}
+	<xmp class=idl>
+		dictionary StartViewTransitionOptions {
+			UpdateCallback? update = null;
+			sequence<DOMString>? types = null;
+		};
 
-		1. If |inboundViewTransitionParams| is null, then return null.
+		partial interface Document {
 
-		1. Set |document|'s [=inbound view transition params=] to null.
+			ViewTransition startViewTransition(optional (UpdateCallback or StartViewTransitionOptions) callbackOptions = {});
+		};
+	</xmp>
 
-		1. If |document|'s [=active view transition=] is not null, then return null.
 
-			Note: this means that starting a same-document transition before revealing the document would cancel a pending cross-document transition.
+	<div algorithm="start-vt-with-options">
+		The [=method steps=] for <dfn method for=Document>startViewTransition(|callbackOptions|)</dfn> are as follows:
 
-		1. [=Resolve @view-transition rule=] for |document| and let |resolvedRule| be the result.
+		1. Let |updateCallback| be null.
 
-		1. If |resolvedRule| is "<code>skip transition</code>", then return null.
+		1. If |callbackOptions| is an an {{UpdateCallback}}, set |updateCallback| to |callbackOptions|.
 
-		1. Let |transition| be a new {{ViewTransition}} in |document|'s [=relevant Realm=],
-			whose [=ViewTransition/named elements=] is |inboundViewTransitionParams|'s [=view transition params/named elements=],
-			and [=ViewTransition/initial snapshot containing block size=] is |inboundViewTransitionParams|'s [=view transition params/initial snapshot containing block size=].
+		1. Otherwise, if |callbackOptions| is a {{StartViewTransitionOptions}}, then set |updateCallback| to |callbackOptions|'s {{StartViewTransitionOptions/update}}.
 
-		1. Set |document|'s [=active view transition=] to |transition|.
+		1. If |this|'s [=active view transition=] is not null and its [=outbound post-capture steps=] is not null,
+			then:
 
-		1. [=Resolve=] |transition|’s [=ViewTransition/update callback done promise=] with undefined.
+			1. Let |preSkippedTransition| be a new {{ViewTransition}} in |this|'s [=relevant realm=] whose [=ViewTransition/update callback=] is |updateCallback|.
 
-		1. Set |transition|’s [=ViewTransition/phase=] to "`update-callback-called`".
+			1. [=Skip the view transition|Skip=] |preSkippedTransition| with an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
We're using InvalidStateError here but when skipping while setting up the cross-doc transition on the old Doc we use AbortError. That's intentional right? Because in the latter case we abort an already started transition.